cmd/stolon-pgbouncer/main.go

```diff
@@ -490,6 +512,12 @@ func main() {
 								return nil
 							}
 
+							// Let the pgbouncer.child know we've got a value, and that it's ok to boot
+							// PgBouncer. We'll probably fail to reload PgBouncer immediately after the
+							// first time we run this as we'll race PgBouncer to start, however our
+							// subsequent retry will succeed.
+							signalReceivedKeeperHost()
```

Comment:
Should we call this after we have generated a new config with the masterAddress?

So at line 530, between `pgBouncer.GenerateConfig` and `pgBouncer.Reload`?

We could then get rid of the initial fake config creation at line 370 as we would never start PgBouncer unless we had a masterAddress and already created a config with it.

Reply:
Yeah let's do that